homeassistant/components/systemmonitor/sensor.py

```diff
@@ -334,39 +340,84 @@ async def async_setup_platform(
     discovery_info: DiscoveryInfoType | None = None,
 ) -> None:
     """Set up the system monitor sensors."""
+    processes = [
+        resource[CONF_ARG]
+        for resource in config[CONF_RESOURCES]
+        if resource[CONF_TYPE] == "process"
+    ]
+    hass.async_create_task(
+        hass.config_entries.flow.async_init(
+            DOMAIN, context={"source": SOURCE_IMPORT}, data={"processes": processes}
```

Comment:
I think we should import the config and enable the configured sensors by default. Otherwise existing sensors will get disabled.

Reply:
I think it's better to be safe than sorry. We can remove all that logic when we remove the import after the deprecation period.

Reply:
I made some comments to it in the code so it should be pretty easy for someone to cleanup once deprecation is over